plugins/weather/src/main/resources/WeatherSpeech.aim.xml

```diff
@@ -16,6 +16,18 @@
 		ref="de.unistuttgart.iaas.amyassist.amy.plugin.weather.WeatherSpeechCommand.weatherWeekend">
 		<gram>[how is the] weather [at the] weekend</gram>
 	</Intent>
+	<Intent
+		ref="de.unistuttgart.iaas.amyassist.amy.plugin.weather.WeatherSpeechCommand.rainCheck">
+		<gram>is it gonna rain {timespan} [(at|in) {locationname}]</gram>
+		<EntityTemplates>
+			<EntityTemplate id="timespan" required="false">
+				<gram>(today|tomorrow|at the weekend)</gram>
```

Comment:
`On the weekend` would be the proper term

Reply:
I think `{timespan}` should be at the end of the sentence. Is that possible? Not sure if it conflicts with the wildcard or +

Reply:
Yeah you are right, putting it at the end makes the sentence more natural

Reply:
I agree, i'll try to change it